src/platforms/unity/configuration/app-not-responding.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: App Not Responding (ANR)
+sidebar_order: 20
+description: "Learn how to turn off or specify ANR."
+---
+
+Application Not Responding (ANR) errors are triggered when the Unity thread of the game is blocked for more than five seconds. The Unity SDK reports ANR errors as Sentry events. In addition, Sentry calculates [ANR rate](/platforms/unity/performance/instrumentation/performance-metrics/#application-not-responding-anr-rate) based on these events and user sessions.
+
+## ANR implementation details
+
+The ANR detection in the Unity SDK strictly works within the Unity game, ireespecitve of the targeted platform and works in two parts. First the SDK starts a coroutine that periodically notes the time it last got called. Second, a background thread checks the time since the last update against the ANR threshold from the options.
+The detection is sensitive of the app losing and regaining focus and gets paused accordingly. It is also using `WaitForSecondsRealtime` so pausing your game by modifying the `Time.timeScale` will not trigger false ANR reports.
+
+On platforms that do no support multithreading the ANR detection falls back to relying on the coroutine itself to report if the last time it executed exceeds the timeout.
```

Comment:
```suggestion
Detection gets paused occasionally to avoid having the app repeatedly lose and regain focus. It also uses `WaitForSecondsRealtime` which makes it possible to pause a game by modifying the `Time.timeScale` without triggering false ANR reports.

If a platform doesn't support multithreading, ANR detection is dependent on the coroutine itself reporting when execution exceeded the ANR timeout threshold.
```
Again, let me know if I'm misunderstanding.

Reply:
This is supposed to describe a mechanism that pauses and unpauses the ANR detection with the app. 
If you minimize your game or you switch between apps on your phone there won't be any false positives.
It's a "Detection is aware of the app getting paused or losing focus and does so too." thing.